sass/inputs.sass

```diff
@@ -0,0 +1,6 @@
+input[type=radio]
+  margin: 0.5rem
+
+
+input[type=radio]:checked 
+  accent-color: #7967e1
```

Comment:
Can you replace this with the `$primary` variable from [vars.sass](/neuralbertatech/natUI/blob/master/sass/vars.sass). This way if we change the value of the primary colour, the changes will take effect here.